core/trino-parser/src/test/java/io/trino/sql/parser/TestSqlParser.java

```diff
@@ -1081,6 +1094,52 @@ public void testShowFunctions()
         assertStatement("SHOW FUNCTIONS LIKE '%' ESCAPE '$'", new ShowFunctions(Optional.of("%"), Optional.of("$")));
     }
 
+    @Test
+    public void testTrimBuiltInFunction()
```

Comment:
I don't think we need to test them in a query here. Instead, we might test how the new syntax works in `TestStringFunctions`.